website/content/configuration/troubleshooting.md

```diff
@@ -67,3 +67,52 @@ In addition to the above, make sure to watch the logs of MetalLB's speaker compo
 $ kubetail -l component=speaker -n metallb-system
 ...
 ```
+
+## Status Endpoint
```

Comment:
I don't see this as an API but rather as a very primitive UI for getting some insight into what MetalLB is doing. I've added the status endpoint following Dave's suggestion. The rationale is that with peer autodiscovery added it becomes trickier to figure out why a specific peer exists/doesn't exist since you now have multiple sources for peer configuration.

Reply:
I've expanded the note in this section of the docs to warn users that the endpoint is intended for debugging by humans and not for automation.

Reply:
Perfect, I support this not being an API, just wanted to make sure you agree and we document it :)